content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -3,126 +3,122 @@ title: Impact of the CircleCI Security Incident on the Datadog Agent
+If either one of these is identified as trusting the key, take the actions listed in the following sections to secure the affected host. If neither of these is identified as trusting the key, no further action is needed.
 
-* You are on an RPM-based Linux distribution (RHEL, CentOS, Rocky Linux, AlmaLinux, Amazon Linux, SUSE/SLES, Fedora), **and** your system trusts the affected GPG key
+1. Check the RPM database by running the following command:
 
-## Does my system trust the affected key?
+   ```bash
+   $ rpm -q gpg-pubkey-4172a230-55dd14f6
+   ```
 
-Two places need to be checked to verify if your system trusts the affected key: the RPM database and the Datadog repo file. If either one of these is identified as trusting the key, we recommend taking the actions listed in the following sections. If neither of these is identified as trusting the key, no further action is needed.
+   **The host trusts the key and requires action** if the command runs successfully and prints `gpg-pubkey-4172a230-55dd14f6`.**The host does not trust the key in the RPM database** if the command fails, exiting with a non-0 exit code and outputting a message like `package gpg-pubkey-4172a230-55dd14f6 is not installed`.
 
-### Verifying if the key is imported in the RPM database
+2. Check the Datadog repo file. The default location for the file is:
 
-Run the following command:
+   - RHEL, CentOS, Rocky Linux, AlmaLinux, Amazon Linux, and Fedora: `/etc/yum.repos.d/datadog.repo`
+   - OpenSUSE and SLES:  `/etc/zypp/repos.d/datadog.repo`
 
-```bash
-$ rpm -q gpg-pubkey-4172a230-55dd14f6
-```
+   If the repo file mentions `DATADOG_RPM_KEY.public` under the `gpgkey` entry as shown in the following examples, **the host trusts the affected key and requires action**:
 
-* If the command runs successfully and prints `gpg-pubkey-4172a230-55dd14f6`, your system trusts the affected key.
-* If the command fails (exiting with a non-0 exit code and outputting a message like `package gpg-pubkey-4172a230-55dd14f6 is not installed`), your system does not trust the affected key.
+   * `https://s3.amazonaws.com/public-signing-keys/DATADOG_RPM_KEY.public`
+   * `https://keys.datadoghq.com/DATADOG_RPM_KEY.public`
+   * `https://s3.amazonaws.com/yum.datadoghq.com/DATADOG_RPM_KEY.public`
+   * `https://yum.datadoghq.com/DATADOG_RPM_KEY.public`
 
-### Verifying if the key is used in the Datadog repo file
 
-For default installations, the Datadog repo file can be found at:
+### Prioritizing your search
 
-* `/etc/yum.repos.d/datadog.repo` on RHEL, CentOS, Rocky Linux, AlmaLinux, Amazon Linux, and Fedora
-* `/etc/zypp/repos.d/datadog.repo` on OpenSUSE and SLES
+Because of the variety of ways packages can be installed and updated over time, Datadog recommends you check **all** RPM-based Linux hosts with a Datadog package installed. To help you prioritize which hosts to check first, consider the following guidance. The following scenarios are **more likely** to be vulnerable:
```

Comment:
```suggestion
Because of the variety of ways packages can be installed and updated over time, Datadog recommends you check **all** RPM-based Linux hosts with a Datadog package installed. To help you prioritize which hosts to check first, consider the following guidance. The following scenarios are **more likely** to be affected:
```